### DISCUSSION TITLE
How to whitelist a package to bypass rate limit.

### DISCUSSION THREAD

**Discussion by sppatel** [renovatebot/renovate#12699] [Request Help]
We have a global rate limit across all repositories in our organization to limit the number of open PRs to 5 for a given repo.  However, we want to whitelist certain packages such that if there is a new version, always open the PR even if this exceeds the limit.

I thought this adding a package rule with the following would do it - but does not appear to work and the package is still stuck behind rate limiting per the dependency dashboard.

`extends: [':prConcurrentLimitNone', ':prHourlyLimit1'],
`

Is there a way to do this?

**Comment by rarkins** [comment]
Currently rate limiting is shared, and not possible to bypass on a per-package basis

**Comment by AurimasNav** [comment]
I would love to bypass limit for private registries.
